assets/js/source_lv_hooks.js

```diff
@@ -254,4 +254,22 @@ hooks.FormatTimestamps = {
   },
 };
 
+hooks.DocumentVisibility = {
+  mounted() {
+    this.handleVisibilityChange = () => {
+      this.pushEvent("visibility_change", {
+        visibility: document.visibilityState,
```

Comment:
i think we will also need to consider document blur/focus, as i notice that when the window goes to the background, the polling still continues as the visibilityState api does not consider unminimized unfocused windows as "non-visible".

can be done in separate PR.